### DISCUSSION TITLE
Not able to read the miniO and S3 data

### DISCUSSION THREAD

**Discussion by Raghavendra-Mankar** [grafana/mimir#9255] [Development]
Hi,
I'm not able to read the data from miniO and S3 data getting following error and please find mimir.yaml below, please help here i tried both miniO and S3 getting same error in both storage.

"expanding series: failed to fetch some blocks (err-mimir-store-consistency-check-failed). The failed blocks are: 01J7DVMQKNMHP6QABDSV132KHK 01J7DVPJ6KGDB5S97G104V18QA 01J7DVT7C60CDK9J1W7MVD8TH6 01J7DVXWJ4NWSD5GHX2ZWT2XST"

 
  #----Latest mimir.yaml
server:
  http_listen_port: 9009
  log_level: debug

multitenancy_enabled: false

compactor:
  compaction_interval: 10m  # Less frequent compaction
  first_level_compaction_wait_period: 10m

distributor:
  pool:
    health_check_ingesters: true
  ring:
    instance_addr: 0.0.0.0
   
ingester:
  ring:
    final_sleep: 0s
    instance_addr: 0.0.0.0
    num_tokens: 512
    replication_factor: 1  # Increase for redundancy

store_gateway:
  sharding_ring:
    replication_factor: 1
    instance_addr: 0.0.0.0
    num_tokens: 512
    zone_awareness_enabled: false  # Disable if not using zones

blocks_storage:
  backend: s3
  s3:
    endpoint: "192.168.1.2:9000"  # MinIO API endpoint
    bucket_name: "mimirmetricsdata"        # The bucket you created
    access_key_id: "mI8KRIncz4oEFjNT8Vfo"   # MinIO access key
    secret_access_key: "ywg8MP4NDsPAWQu17yP8SG295BdfFUOcgpW52R2u" # MinIO secret key
    insecure: true     

  tsdb:
    ship_interval: 10m
    block_ranges_period: [10m]
    retention_period: 720h
    head_compaction_interval: 10m
    ship_concurrency: 20
  bucket_store:
    sync_interval: 15m
    block_sync_concurrency: 50  # Match store-gateway concurrency

querier:
  query_store_after: 15m  # Increase to allow querying ingesters first
  max_query_into_future: 20m
  promql_engine: true

ingester_client:
  circuit_breaker:
    enabled: true
    
flusher:
  exit_after_flush: true

**Comment by 56quarters** [comment]
You've changed quite a few advanced TSDB and query settings. These are advanced settings and are not meant be change without a deep understanding of how Mimir works. Please use the default values to avoid this issue.

**Reply by Raghavendra-Mankar** [reply]
Thanks for the reply.
I used the default values now, but getting same error, Is there any other configuration i missed, because I'm able to write the data properly, After some time not able to query the data from s3 or miniO.
"expanding series: failed to fetch some blocks (err-mimir-store-consistency-check-failed). The failed blocks are: 01J7DVMQKNMHP6QABDSV132KHK 01J7DVPJ6KGDB5S97G104V18QA 01J7DVT7C60CDK9J1W7MVD8TH6 01J7DVXWJ4NWSD5GHX2ZWT2XST"
